crates/cargo-winrt/src/main.rs

```diff
@@ -368,6 +379,27 @@ USAGE:
     }
 }
 
+#[derive(Debug)]
+struct Version {}
+
+impl Version {
+    fn perform(&self) -> anyhow::Result<()> {
+        println!("cargo-winrt {}", VERSION);
+        Ok(())
+    }
+
+    fn print_help() {
+        println!(
+            r#"
```

Comment:
Just thinking out loud, I wonder if it would be a good idea to put all these help messages in text files and include them using `include!`...

Reply:
It's definitely ugly the way it is, but I also think there are advantages to keeping the help text with the command, in that you're more likely to see it and update it when updating the command. Though I suspect any conversation about command ergonomics is all a little bit pedantic, because it seems like the number of commands that would theoretically be added once things reach a steady state is pretty low?